README.md

```diff
@@ -1,13 +1,45 @@
-# ShyBye
+# ShyBye #
+**Our mission** is to be a place for individuals overcome social anxiety and
+loneliness by gamifying primarily in-person social interactions, providing a
+variety of fun social challenges
 
-ShyBye is an app that aims to help people reduce their social anxiety by engaging them in social challenges.
+ShyBye was initially created over the span of *four days* during the 2022
+"[GDI Virtual Hackathon](https://girldevelopit.com/virtual-hackathon/): Hack for
+Health!" and won the "On Our Phone Tomorrow!" award
 
-- Frontend: React + JS + MUI
-- Backend: Spring + Java + Postgres
+Made with &hearts; by all [our awesome contributors](https://shybye.netlify.app/about)!
+<!-- TODO: add a contributor section -->
 
-See the backend code here: https://github.com/Valorieb/shybye-backend
+## Under the Hood ##
+This is the frontend source code for the ShyBye app
+- [**Frontend**](https://github.com/gabw13/shybye-frontend)
+[Next.js](https://nextjs.org) + [MUI](https://mui.com/core/)
+- [**Backend**](https://github.com/Valorieb/shybye-backend)
```

Comment:
are these links still necessary, since we now have the repos under the shybye org?

Reply:
For now I'd say so since it makes it easier to jump back-and-forth.
Until we consolidate everything into one repo and determine our deployment strategy I think it's alright